spec/unit/network/authorization_spec.rb

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env ruby
 require 'spec_helper'
+require 'puppet/network/http'
```

Comment:
I only added these requires because without them, it wasn't possible to run the `authorization_spec` tests in isolation.  Without these, you'd get an `undefined Puppet::Network::Authorization` error when trying to run `bundle exec rake spec TEST=spec/unit/network/authorization_spec.rb`.